server/service/hosts.go

```diff
@@ -924,11 +924,12 @@ func (svc *Service) getHostDetails(ctx context.Context, host *fleet.Host, opts f
 		switch host.Platform {
 		case "windows":
 			if ac.MDM.WindowsEnabledAndConfigured && license.IsPremium(ctx) {
-				bls, err := svc.ds.GetMDMWindowsBitLockerStatus(ctx, host)
+				hde, err := svc.ds.GetMDMWindowsBitLockerStatus(ctx, host)
 				if err != nil {
```

Comment:
That's what I was referring to earlier - in case of error we don't even look at `hde` so typically we return a nil value along with the error. Not critical though, I know this is needed ASAP so that's fine to leave like that.